tests/models/mamba2/test_modeling_mamba2.py

```diff
@@ -134,10 +146,28 @@ def get_pipeline_config(self):
         config = self.get_config()
         config.vocab_size = 300
         return config
+    
+    def prepare_config_and_inputs_for_decoder(self):
+        (
+            config,
+            input_ids,
+            sequence_labels,
+            token_labels,
+            choice_labels,
+        ) = self.prepare_config_and_inputs()
+
+        return (
+            config,
+            input_ids,
+            sequence_labels,
+            token_labels,
+            choice_labels,
+        )
 
     def create_and_check_mamba2_model(
-        self, config, input_ids, input_mask, sequence_labels, token_labels, choice_labels
+        self, config, input_ids, input_mask, *args
```

Comment:
Maybe undo this, I'm not entirely sure why but the tests are broken in comparison to main.